share/templates/08-Orchestral/02-Symphony_Orchestra.mscx

```diff
@@ -957,6 +955,7 @@
           </Articulation>
         <Channel>
           <controller ctrl="0" value="1"/>
+          <controller ctrl="32" value="0"/>
```

Comment:
I'm not sure about those added `controller` tags, not sure whether thay harm or not

Reply:
These were added automatically, I don't know what they mean...

Reply:
IIRC `ctrl=32` is the expression controller, for single note dynamics

Reply:
So value="0" suggests there's no single note dynamics?

Reply:
Probably not, I see other values increasing for ctrl="32"...

Reply:
I don't know what thoer `value` settings mean, that's why I'd shy away from adding those tags.

Reply:
In the original file ctrl="0" tags are missing in some places and ctrl="32" tags missing in some other places, I guess MuseScore just completed what're lost.

Reply:
BTW and in case you were not aware: there is a special `--template-mode` command line option

Reply:
Where and how to execute it?

Reply:
...MuseScore3.exe --template-mode

Reply:
Still didn't get it

Reply:
See https://musescore.org/en/handbook/3/command-line-options

Reply:
It's the MSB for the bank select (i.e. the big half of the binary number that decides what midi instrument we use). The default is 0, which is why it hadn't been written before. For some reason it's still being written. Actually, I might know why. Did you manually change any of the patches in the mixer, Howard?

Reply:
OK, so it is at least harmless and nothing that hinders this PR from getting merged

Reply:
> It's the MSB for the bank select (i.e. the big half of the binary number that decides what midi instrument we use). The default is 0, which is why it hadn't been written before. For some reason it's still being written. Actually, I might know why. Did you manually change any of the patches in the mixer, Howard?

I didn't, just change the pitch range of contrabasses, save, close.

Reply:
OK, so we all agree that the added tags `ctrl="0"` and `ctrl="32"` not harmful, right?

Reply:
Yes, they are harmless

Reply:
Very well then :D

Reply:
Hmmm. Maybe not entirely harmless, actually. If a bank number is written, we don't try to switch the instrument to an SND-enabled version. That means that SND might not end up being enabled for the template...

Reply:
> Hmmm. Maybe not entirely harmless, actually. If a bank number is written, we don't try to switch the instrument to an SND-enabled version. That means that SND might not end up being enabled for the template...

Don't quite understand things about SND here XD

Reply:
Whatever: just drop those added controller tags from your PR and it should be fine in any case

Reply:
Done.